README.md

```diff
@@ -76,13 +76,13 @@ where app.json may look something like:
 }
 ```
 
-You can specify a specific node_modules top level path by using the `-p` flag.
+You can specify a specific path to be passed to Glue as the `relativeTo` option by using the `-p` flag.
 
 ```javascript
-rejoice -c app.json -p /full/path/node_modules
+rejoice -c app.json -p /full/path/to/project/plugin/dir
```

Comment:
I think we still want to keep node_modules here right?  Without the -r flag this doesn't append a node_modules directory like the -r context.